sass/_getting-started.scss

```
@@ -0,0 +1,314 @@
+
+  // Tab content visibility
+  .gs-tab-content {
+    display: none;
+    animation: gsFadeIn 0.3s ease;
+
+    &.gs-active {
+      display: block;
+    }
+  }
+
+  @keyframes gsFadeIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "gsFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)